src/components/ui/ColorButtons.scss

```diff
@@ -0,0 +1,33 @@
+.colorbuttons{
+  position: fixed;
```

Comment:
position fixed for all buttons?
Why?

Reply:
probably something i forgot to remove when i copied some code around, deleted it